Gruntfile.js

```diff
@@ -1567,8 +1567,6 @@ module.exports = function(grunt) {
 	grunt.registerTask( 'verify:source-maps', function() {
```

Comment:
@c4rl0sbr4v0 Since we took the time to understand why we care about this, let's add a descriptive comment to the preceding comment, something like:

> Compiled JavaScript files may link to sourcemaps. In some cases, the source map may not be available, which can cause 404 errors when browsers try to download the sourcemap from the referenced URLs. Ensure that sourcemap links are not included in JavaScript files.

Reply:
Done. Great work finding out @sirreal !